docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
+* can type relatively fast while maintaining accuracy
+* prefers being on desktop rather than on mobile
```

Comment:
For this, I am not very sure how applicable is the "prefers desktop over mobile" part - because usually in clinics, management of databases etc. is done on computers, no? (At least for private clinics / polyclinics that I have visited). So it's not a "preference", but rather a requirement. @Aquarinte maybe you can enlighten us on this? :)

Reply:
From what I have observed so far, both Vets and Vet technicians use desktops. They have to deal with a lot of information (e.g. appointments, medical histories, patient registrations etc), so desktop and physical keyboard are a more viable option compared to mobile.

Reply:
So I guess I will change this to "Uses a desktop for work"?